packages/tools/client-debugger/client-debugger-view/src/components/ContainerHistoryView.tsx

```diff
@@ -80,105 +88,69 @@ export function ContainerHistoryView(props: ContainerHistoryProps): React.ReactE
-						? "#FDF5E6" // yellow
-						: containerHistory[i].newState === ContainerStateChangeKind.Closed
-						? "#FFF0F5" // red
-						: containerHistory[i].newState === ContainerStateChangeKind.Disposed
-						? "#FFE4E1" // dark red
-						: "#C0C0C0", // grey for unknown state
-				borderStyle: "solid",
-				borderWidth: 1,
-				borderColor: DefaultPalette.neutralTertiary,
-				padding: 3,
-			},
-		};
-
-		const iconStyle: IStackItemStyles = {
-			root: {
-				padding: 10,
-			},
-		};
-
-		historyViews.push(
-			<div key={`container-history-info-${i}`}>
-				<Stack horizontal={true} styles={accordionBackgroundColor}>
-					<StackItem styles={iconStyle}>
-						<Icon
-							iconName={
-								containerHistory[i].newState === ContainerStateChangeKind.Connected
-									? "PlugConnected"
-									: containerHistory[i].newState ===
-									  ContainerStateChangeKind.Attached
-									? "Attach"
-									: containerHistory[i].newState ===
-									  ContainerStateChangeKind.Disconnected
-									? "PlugDisconnected"
-									: containerHistory[i].newState ===
-									  ContainerStateChangeKind.Disposed
-									? "RemoveLink"
-									: containerHistory[i].newState ===
-									  ContainerStateChangeKind.Closed
-									? "SkypeCircleMinus"
-									: "Help"
-							}
-						/>
-					</StackItem>
-					<StackItem>
-						<div
-							key={`${containerHistory[i].newState}-${containerHistory[i].timestamp}`}
-						>
-							<b>State: </b>
-							{containerHistory[i].newState}
-							<br />
-							<b>Time: </b>
-							{wasChangeToday
-								? changeTimeStamp.toTimeString()
-								: changeTimeStamp.toDateString()}
-							<br />
-						</div>
-					</StackItem>
-				</Stack>
-			</div>,
-		);
-	}
+	// Columns for rendering audience history
+	const containerHistoryColumns = [
+		{ columnKey: "state", label: "State" },
+		{ columnKey: "time", label: "Time" },
+	];
+
+	const getBackgroundColorForState = (state: string): string => {
+		switch (state) {
+			case "connected":
+				return tokens.colorPaletteGreenBackground2; // green
+			case "disconnected":
+				return tokens.colorPaletteDarkOrangeBorderActive; // orange
+			case "closed":
+				return tokens.colorPaletteRedBorder1; // red
+			case "disposed":
+				return tokens.colorPaletteDarkRedBackground2; // dark red
+			case "attached":
+				return tokens.colorPaletteRoyalBlueBackground2; // blue
+			default:
+				console.log("Unknown state type for container!");
+				return tokens.colorBrandBackgroundPressed; // black
+		}
+	};
 
 	return (
-		<Stack
-			styles={{
-				root: {
-					height: "100%",
-				},
-			}}
-		>
-			<StackItem>
-				<h3>Container State History</h3>
-				<Stack
-					styles={{
-						root: {
-							overflowY: "auto",
-							height: "300px",
-						},
-					}}
-				>
-					<div style={{ overflowY: "scroll" }}>{historyViews}</div>
-				</Stack>
-			</StackItem>
-		</Stack>
+		<>
+			<Divider appearance="brand"> Audience State </Divider>
```

Comment:
It also might be better to move the Divider bar up to the parent component, rather than having each child view render one.